Zend/zend_compile.h

```diff
@@ -555,6 +558,7 @@ struct _zend_execute_data {
 #define ZEND_CALL_OBSERVED           (1 << 28) /* "fcall_begin" observer handler may set this flag */
                                                /* to prevent optimization in RETURN handler and    */
                                                /* keep all local variables for "fcall_end" handler */
+#define ZEND_CALL_PARTIAL            (1 << 29)
```

Comment:
This isn't currently used, the frame is marked in case it needed any special handling ... it doesn't seem necessary, will drop if not required.